package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@launchpadlab/prettier-config",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "main": "index.json",
   "repository": "https://github.com/launchpadlab/prettier-config",
   "author": "dpikt",
   "license": "MIT",
+  "files": ["index.json"],
```

Comment:
Confirming we don't need `README` or `package.json`?

Reply:
Correct, `npm` adds that automatically 🙂 . The screenshot above shows them in the "Tarball Contents" section